tests/_convert/test_ipynb.py

```diff
@@ -518,24 +495,6 @@ def test_transform_duplicate_definitions_with_reference_to_previous():
     ]
 
 
-def test_transform_cell_metadata_with_complex_metadata():
-    sources = [
-        "print('Complex metadata')",
-    ]
-    metadata = [
-        {
-            "tags": ["tag1", "tag2"],
-            "collapsed": True,
-            "scrolled": False,
-            "custom": {"key": "value"},
-        }
-    ]
-    result = transform_cell_metadata(sources, metadata)
-    assert result == [
-        "# Cell tags: tag1, tag2\nprint('Complex metadata')",
-    ]
-
-
```

Comment:
I moved these tests to `cell_metadata.ipynb` input, with the associated snapshot. I think it will be easier to test the expected outputs that way in the future (just updating the snapshot, potentially for the same tags/metadata as we better adapt).